scripts/tcs_listener.sh

```diff
@@ -22,4 +22,4 @@ listener_port=`grep listener_port ${TCF_HOME}/config/tcs_config.toml | awk {'pri
 port=${TCF_TCS_LISTENER_PORT:-${listener_port}}
 
 echo "starting TCS listener ..."
-python3 $listener --bind_uri ${port}
+python3 $listener --bind_uri ${port} --lmdb_url http://avalon-lmdb:9090
```

Comment:
We don't need bind_port. Port value is already present in lmdb_url. We can extract port from url

Reply:
I will remove lmdb_url from config and mandate to use command line argument for listener and EM

Reply:
@Ram-srini Don't remove this url from lmdb_config.toml. In my opinion both options should be available. If this is passed as command line argument, it will take the precedence.